docs/TargetMaintenance/CreatingAUnifiedTarget.md

```diff
@@ -44,7 +44,9 @@ These instructions explain how to create a Unified Target configuration for an e
 
 - connect to your board running the Unified Target firmware, enter CLI;
 
-- copy / paste the contents of the dump created in 2. into CLI, then enter `save`;
+- copy / paste the contents of the dump created in 2. into CLI
+
+- enter the command: `feature OSD` in CLI, just before the #master section (to switch on OSD by default), then enter `save`;
```

Comment:
Not sure why the instructions should be to enter this here, and then add all other custom settings in 6. below - this does not seem to make sense.

Reply:
@mikeller - was following your description on #Slack - it said to enter it under step 5 :-)
![image](https://user-images.githubusercontent.com/12601105/67184450-14e29f00-f3e4-11e9-8df5-67cdd84ae2d9.png). Will change it though.

Reply:
Heh, shows the danger of referencing stuff by index. What happened is that #9012 was merged after I posted on Slack, and the numbering of the steps in the document were shifted by 1. :grimacing: